### PR TITLE
Busser dev

### PR DESCRIPTION
**Pull request by gauravj09**

### CODE REVIEW

File: backend_models/busser.php [hunk 1]
```diff
@@ -1,14 +1,49 @@
...
+
+			if (count($unclean_tables) > 0)
+			{
+				return $unclean_tables;
+			}
+
+			return -1;
+		}
+
+		function setTableStateOpen()
```
Comment (Ajinkya-Nawarkar) [inline backend_models/busser.php]: Well so here what's happening is that all tables are getting set to open at once. What you wanna do is take in an index or TID (depends on how you want it) and then set that specific table as open and not all.

Comment (Ajinkya-Nawarkar) [reply backend_models/busser.php]: check the busser's frontpage again

Comment (gauravj09) [reply backend_models/busser.php]: > Well so here what's happening is that all tables are getting set to open at once. What you wanna do is take in an index or TID (depends on how you want it) and then set that specific table as open and not all.

Can you clarify on this comment? I thought the front end needed a list of unclean tables (which I gave them in the first method) Then I check the list of unclean tables using TID (like you said) and update their status one by one in the setTableStateOpen().

Comment (Ajinkya-Nawarkar) [reply backend_models/busser.php]: Have you viewed the busser.php yet? The way it's set up right now is every table has a "clean now" button. In which case, you'd have to maintain a list of table_objects and only set specific tables as open based on what TID or index is passed to you from the user.

**Comment by Ajinkya-Nawarkar** [comment]
Will have to also resolve the conflicts before reviewing and merging

**Comment by gauravj09** [comment]
Yeah working on it right now

**Pull request merged**